<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWidget</class>
 <widget class="QWidget" name="loginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>464</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>464</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>652</width>
    <height>464</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>loginWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="signlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>icon</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="registerpushButton">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>425</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
color: rgb(9, 50, 255);</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginpushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入您的密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入您的邮箱</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
</string>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);
border-style:outset;
border-width:0px;
border-color:rgb(255,0,0);
border-radius:10px;

padding:1px;</string>
   </property>
   <property name="text">
    <string>QiuQiu</string>
   </property>
  </widget>
  <widget class="QLabel" name="giflabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>220</y>
     <width>671</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-width:0px;
border-radius:50px;
border-style:solid;
border-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:0px;
border-radius:40px;
border-image: url(:/image/toolicon/usericon.jpg);
border-style:solid;
border-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/image/toolicon/reduce-one.png</normaloff>:/image/toolicon/reduce-one.png</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/image/toolicon/close-one.png</normaloff>:/image/toolicon/close-one.png</iconset>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFontComboBox" name="idfontComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>giflabel</zorder>
  <zorder>idfontComboBox</zorder>
  <zorder>pwlineEdit</zorder>
  <zorder>signlabel</zorder>
  <zorder>registerpushButton</zorder>
  <zorder>loginpushButton</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>toolButton</zorder>
  <zorder>toolButton_2</zorder>
  <zorder>idlineEdit</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
